snowball.php

```diff
@@ -376,11 +373,13 @@ function snowball_save_block($json_block, $post_id) {
   return $was_successful;
 }
 
-// if successful return insert_id
-// if fails return -1
-// TODO: error checking beginning and after the function.
 function snowball_save_article($article, $post_id) {
   global $wpdb;
+  $nonce = $_POST['snowball_ajax_nonce'];
+
+  if (! wp_verify_nonce( $nonce, 'snowball_ajax_nonce')) {
```

Comment:
Let's delete the space after `!`. Not sure if it makes a difference, but just to be safe & consistent.

Also can delete space before `$nonce`.